types/v56/types.go

```diff
@@ -2240,31 +2242,40 @@ type QueryResultEdgeGatewayRecordType struct {
 // QueryResultVMRecordType represents a VM record as query result.
 type QueryResultVMRecordType struct {
 	// Attributes
-	HREF                    string `xml:"href,attr,omitempty"`       // The URI of the entity.
-	Name                    string `xml:"name,attr,omitempty"`       // VM name.
-	Deployed                bool   `xml:"isDeployed,attr,omitempty"` // True if the virtual machine is deployed.
-	Status                  string `xml:"status,attr,omitempty"`
-	Busy                    bool   `xml:"isBusy,attr,omitempty"`
-	Deleted                 bool   `xml:"isDeleted,attr,omitempty"`
-	MaintenanceMode         bool   `xml:"isInMaintenanceMode,attr,omitempty"`
-	Published               bool   `xml:"isPublished,attr,omitempty"`
-	VAppTemplate            bool   `xml:"isVAppTemplate,attr,omitempty"`
-	VdcEnabled              bool   `xml:"isVdcEnabled,attr,omitempty"`
-	VdcHREF                 string `xml:"vdc,attr,omitempty"`
-	VAppParentHREF          string `xml:"container,attr,omitempty"`
-	VAppParentName          string `xml:"containerName,attr,omitempty"`
-	HardwareVersion         int    `xml:"hardwareVersion,attr,omitempty"`
-	HighestSupportedVersion int    `xml:"pvdcHighestSupportedHardwareVersion,attr,omitempty"`
```

Comment:
Difficult to see in the diff, as formatting changed. Is the only change here that `HighestSupportedVersion` was removed from API in 29.0?

Reply:
No, third of values were added and a few removed.

Reply:
It was interesting - in general a few elements needed to be deleted cause won't be any info in them. Other were added.

Reply:
Please elaborate/give example. You mean API v29.0 defines a field, but returns nothing in it?

Reply:
Then we move from one version to another - API won't return values even we would have property left. So no values to have them.

Reply:
I guess we just need a list of it in changelog for those users who may rely on it

Reply:
added